volume_types.go

```diff
@@ -202,7 +202,7 @@ type VolumeModify struct {
 	//  Size must be a multiple of 8192.
 	Size int64 `json:"size,omitempty"`
 	// Unique identifier of the protection policy assigned to the volume.
-	ProtectionPolicyID string `json:"protection_policy_id,omitempty"`
+	ProtectionPolicyID *string `json:"protection_policy_id,omitempty"`
```

Comment:
@AkshaySainiDell To detach the protection policy from the volume, we need to pass empty string in the payload. If we use simple variable instead of pointer, 'omitempty' removes the attribute from the payload. As, pointer accepts the empty string this change was made.

Reply:
@prajwalpatil25 How about without omitempty? Would it solve the issue?

Reply:
@adarsh-dell made suggested change. thanks

Reply:
Thank you @prajwalpatil25 This looks good to me.